### PR TITLE
Release v0.8.1

### PR DESCRIPTION
**Pull request by ka3de**
- [x] Update version to `v0.8.1` in the `browser/module.go` file.
- [x] Add release notes for the version.
- [x] Update the release notes with the release details.
- [x] Leave the bump commit and squash the rest into `Add v0.8.1 release notes` commit with co-authors as the team members.

### CODE REVIEW

File: release notes/v0.8.1.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This is a patch release that fixes a few bugs, including one [regression bug](https://github.com/grafana/xk6-browser/issues/749) introduced in v0.8.0.
+
+
+## Bugs fixed
+
+- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
+- Fix `Page.URL()` method. [#758](https://github.com/grafana/xk6-browser/pull/758)
+- Fix `Frame.Title()` method. [#761](https://github.com/grafana/xk6-browser/pull/761)
```
[the remark this answers is not in the document]
Comment (ka3de) [reply release notes/v0.8.1.md:9]: See an attempt on that in [9e5af24](https://github.com/grafana/xk6-browser/pull/775/commits/9e5af2453f73afb620c9e77563cfbc6af0c4555f).

File: release notes/v0.8.1.md [hunk 2]
```diff
@@ -4,5 +4,17 @@ This is a patch release that fixes a few bugs, including one [regression bug](ht
 ## Bugs fixed
 
 - Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
+- Fix usage of stale virtual user context in mapping layer. [#754](https://github.com/grafana/xk6-browser/pull/754).
```
Comment (ankur22) [inline release notes/v0.8.1.md:7]: Not sure if anyone was impacted by this, so maybe best to place it in internal?

Comment (ka3de) [reply release notes/v0.8.1.md:7]: Would we keep the wording in that case? E.g.: `Fix...`?

File: release notes/v0.8.1.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+This is a patch release that fixes a few bugs, including one [regression bug](https://github.com/grafana/xk6-browser/issues/749) introduced in v0.8.0, adds one improvement on documentation, and does minor internal modifications.
```
[the remark this answers is not in the document]
Comment (ka3de) [reply release notes/v0.8.1.md:1]: I'm not sure about this change. I followed the format that we established in the previous releases, introducing with a brief summary paragraph and then providing the lists. But I don't have strong opinion about it, just mentioning the reason of why I decided to write it like this :smile:

[the remark this answers is not in the document]
Comment (ankur22) [reply release notes/v0.8.1.md:1]: Happy to leave it as it is now :)

File: release notes/v0.8.1.md (hunk 2 above)
Comment (ka3de) [reply release notes/v0.8.1.md]: Done in [7716840](https://github.com/grafana/xk6-browser/pull/775/commits/77168406e1124f202ccbcd05e1210b22e74a51a8).

File: release notes/v0.8.1.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## Bugs fixed
+
+- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
```
Comment (ankur22) [inline release notes/v0.8.1.md:6]: Almost there i think 🙂 

Atm, we put the issue in brackets after a period but not end it with a period:

e.g.

```suggestion
- Fix mapping of Page's Keyboard, Mouse, and Touchscreen properties. ([#751](https://github.com/grafana/xk6-browser/pull/751))
```

Comment (ankur22) [inline release notes/v0.8.1.md:6]:
```suggestion
- Fix mapping of `Page`'s `Keyboard`, `Mouse`, and `Touchscreen` properties. [#751](https://github.com/grafana/xk6-browser/pull/751)
```

Comment (ka3de) [reply release notes/v0.8.1.md:6]: Agree on the periods.
About the brackets, we did not follow that format in the last release, although we did in the previous ones. I prefer using brackets thought, so will change it.

Comment (ka3de) [reply release notes/v0.8.1.md]: Fixed in: [b5994e3](https://github.com/grafana/xk6-browser/pull/775/commits/b5994e37efaa6064d328a025a4673deae8c4b159).

Comment (ka3de) [reply release notes/v0.8.1.md]: Fixed in: [ed6ca59](https://github.com/grafana/xk6-browser/pull/775/commits/ed6ca59561f868b2f599a83823ea6e4a2d6e742e).

Comment (ankur22) [reply release notes/v0.8.1.md]: > About the brackets, we did not follow that format in the last release

🤦 I missed that 😅

**Review by ankur22 — APPROVED** [review]

**Comment by ankur22** [comment]
Nice, LGTM 🙂

**Pull request merged**